packages/bpk-tokens/formatters/bpk.android.xml.js

```diff
@@ -59,8 +59,8 @@ export const tokenTemplate = ({ name, value, type, category }) =>
     type,
   )}</${tagName(type)}>`; // eslint-disable-line max-len
 
-export default tokens => {
-  const { props } = sortTokens(tokens);
+export default result => {
+  const { props } = sortTokens(result.toJS());
```

Comment:
Custom formatters api has changed. `registerFormat` now accepts a function with an ImmutableJS Map instead of a plain ole' object. See https://github.com/salesforce-ux/theo#custom-format-function.